doc/patching-nixpkgs.xml

```diff
@@ -0,0 +1,70 @@
+<chapter xmlns="http://docbook.org/ns/docbook"
+         xmlns:xlink="http://www.w3.org/1999/xlink"
+         xml:id="chap-patching-nixpkgs">
+ <title>Patching nixpkgs</title>
+ <para>
+   Sometimes you want to track a specific branch of nixpkgs (like
```

Comment:
Please avoid the use of `you` when writing documentation

Reply:
Hi Frederik, can I ask why? Do you have a link to more info on this? Is it a nix convention?

I ask because I personally prefer reading this style of documentation, and all style guides I find on Google encourage using the second person.

Reply:
Actually I don't think we had a style guide at the time, but it was the common style at the time of writing and the way of writing I was used to from academia.